VERSION.cmake

```diff
@@ -1,6 +1,6 @@
-SET( VERSION_MAJOR "3")
-SET( VERSION_MINOR "3" )
-SET( VERSION_PATCH "3" )
+SET( VERSION_MAJOR "4")
+SET( VERSION_MINOR "0" )
+SET( VERSION_PATCH "0" )
```

Comment:
does we need to increase so minor or not? this linkings in C++ is still a bit tricky for me, especially when new method is added.

Reply:
The changes are still binary compatible: Just internal behaviour was changed, and there is just one more (and non-virtual) new method in the YUI class. Not sure if we want to bump that .so number every time for that.